cmd/commands/reset.go

```diff
@@ -0,0 +1,47 @@
+package commands
+
+import (
+	"os"
+	"path"
+
+	"github.com/urfave/cli"
+
+	tmcfg "github.com/tendermint/tendermint/config/tendermint"
+	types "github.com/tendermint/tendermint/types"
+)
+
+var UnsafeResetAllCmd = cli.Command{
+	Name:      "unsafe_reset_all",
+	Usage:     "Reset all blockchain data",
+	ArgsUsage: "",
+	Action: func(c *cli.Context) error {
+		return cmdUnsafeResetAll(c)
+	},
+}
+
+func cmdUnsafeResetAll(c *cli.Context) error {
```

Comment:
This looks like copy-paste from tendermint-core. Should reference common code if possible, even if means changing tendermint core to have exportable init-functions